From 9a49287cff3aeaf70fccdcb910a9d2c8fca7ab38 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Mon, 28 May 2018 15:19:02 +0200
Subject: [PATCH 0759/1852] arm64: zynqmp: Remove broken-cd from zcu100-revC

commit 616398c5218b2fd5b80b7ce2a681b64e74fa3a06 from
https://github.com/Xilinx/linux-xlnx.git

Card detect bit was broken on revA and it is working fine with revC
board that's why this property can be removed.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Link: https://lkml.org/lkml/2020/1/9/494
State: upstream (b4582390d5841c3f6817953553fa3ebf523edb91)
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts
index 3e39454afaaa..cd987590cd3c 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu100-revC.dts
@@ -226,7 +226,6 @@
 &sdhci0 {
 	status = "okay";
 	no-1-8-v;
-	broken-cd; /* CD has to be enabled by default */
 	disable-wp;
 };
 
-- 
2.31.1

